driver/terraform.go

```diff
@@ -439,6 +494,22 @@ func (tf *Terraform) initTaskTemplate() error {
 	return nil
 }
 
+// logDepSize logs the watcher dependency size every nth iteration. Set the
+// iterator to a negative value to log each iteration.
+//
+// This is the same as the controller.logDepSize(). TODO: consider consolidating
+func (tf *Terraform) logDepSize(n uint, i int64) {
```

Comment:
I believe this is the same watcher on the controller shared across all of the drivers, logging at at the controller level should be sufficient and this duplicate logging per driver.

Reply:
It is the same watcher as the controller. I hadn't thought of that the controller logDepSize() will encompass any watcher calls in UpdateTask. Thanks! Let me remove it.

Reply:
You're right, it won't encompass any watcher calls. To clarify about what I mean on duplicate: the watcher size is across all templates, and so this would log the same value per driver(task) as the one in the controller, not providing any new information for users.

Reply:
Ah. Gotcha. Thanks for clarifying!